src/include/daos_api.h

```diff
@@ -220,11 +220,149 @@ daos_prop_alloc(uint32_t entries_nr);
 /**
  * Free the DAOS properties.
  *
- * \param[int]	prop	properties to be freed.
+ * \param[in]	prop	properties to be freed.
  */
 void
 daos_prop_free(daos_prop_t *prop);
 
+/**
```

Comment:
We are about to split the daos_api.h into daos_{container,pool,object, ...}.h
To avoid conflict, I would add this new API to src/include/daos_security.h
We should probably also consider moving the ACL definition there and make it opaque (just a blob) to the property.

Reply:
Thanks for the heads up. I'll move the API functions first - we can move the structures in a follow-on patch, if that's okay. This one is already very big.